tests/MoBi.Tests/Core/SBML/ReactionImporterSpecs.cs

```diff
@@ -69,7 +70,7 @@ protected override void Context()
 
       protected override void Because()
       {
-         sut.DoImport(_sbmlModel, new MoBiProject(), _sbmlInformation, new MoBiMacroCommand());
+         sut.DoImport(_sbmlModel, DomainHelperForSpecs.NewProject(), _sbmlInformation, new MoBiMacroCommand());
```

Comment:
I think you can still use this helper just in case you want to configure the Project all the time with some default stuff